Know Before You Send Guide - Hidden (#3402)
braze-inc/braze-docs: _docs/_hidden/private_betas/know_before_send.md

---
nav_title: Before You Send
description: "After visiting our pre-launch guide, refer to this final list of checks or 'gotchas' for Content Cards, email, in-app messages, push, and SMS."
permalink: /know_before_send/
hidden: true
---

> # Know before you send - a channels guide

Reviewer: ```suggestion
# Know before you send: Channels
```

Launch your campaigns and Canvases with confidence! After visiting our [pre-launch guide](https://labplaybooks.braze.com/canvas-playbooks#/subpage/b2rj8), refer to this final list of checks or "gotchas" for Content Cards, email, in-app messages, push, and SMS.

{% alert note %}
While we provide an extensive list of resources for customers to reference pre-send, each channel has individual nuances that continue to grow as we evolve our products. The checks listed below are helpful suggestions, and we recommend thoroughly testing your campaigns and large sends before sending. 
{% endalert %}

## General

> #### Things to check for

Reviewer: ```suggestion
#### Things to check
```
Suggesting to remove "for" from the "Things to check for" headings

- [**API rate limits**](https://braze.com/resources/articles/whats-rate-limiting): Review the Braze API [rate limits]({{site.baseurl}}/api/api_limits/) for your app groups to avoid running into errors. If you are looking to increase your rate limits (and are already batching requests), reach out to your customer success manager. Keep in mind that this process requires lead-time, so plan accordingly.
- [**Necessary frequency capping overrides**]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/rate-limiting/#frequency-capping): There are some campaigns, like transactional messages, that you will want to always reach the user, even if you have already reached their frequency cap (for example, a delivery notification). If you want a particular campaign to override frequency capping rules, you can set this up in the Braze dashboard when scheduling that campaign's delivery by toggling frequency capping off.

> #### Things to know about

Reviewer: ```suggestion
#### Things to know about
```
Suggesting to remove "about" from the "Things to know about" headings

- [**Global control groups**]({{site.baseurl}}/user_guide/engagement_tools/testing/global_control_group#global-control-group): If you are using a global control group, a percentage of users will not receive any campaigns or Canvases. (you can create exceptions with [exclusion settings]({{site.baseurl}}/user_guide/engagement_tools/testing/global_control_group/#step-3-assign-exclusion-settings)). If you would like to see a list of these users, export them via CSV or [API]({{site.baseurl}}/api/endpoints/export/user_data/post_users_global_control_group/).

> - [**Canvas rate limits**]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/rate-limiting/#rate-limiting): In a Canvas, the rate limit applies across the entire Canvas, not the individual steps. For example, if you were to set a 10,000 message/minute rate limit on a Canvas with multiple steps, it will still be limited to 10,000 messages because the limit will have been reached at the first step.

Reviewer: ```suggestion
- [**Canvas rate limits**]({{site.baseurl}}/user_guide/engagement_tools/campaigns/building_campaigns/rate-limiting/#rate-limiting): In a Canvas, the rate limit applies across the entire Canvas, not the individual steps. For example, if you were to set a 10,000 message per minute rate limit on a Canvas with multiple steps, it will still be limited to 10,000 messages because the limit will have been reached at the first step.
```

- **Frequency capping**: 
  - Frequency capping rules will be applied to push, email, SMS, and webhooks, but not to in-app messages and Content Cards

>   - Global frequency capping is scheduled based on the user’s time zone and is calculated by calendar days, not 24-hour periods. For example, if you set up a frequency capping rule of sending no more than one campaign a day, a user may receive a message at 11 pm in their local time zone, and they would be eligible to receive another message an hour later.

Reviewer: ```suggestion
  - Global frequency capping is scheduled based on the user’s time zone and is calculated by calendar days, not 24-hour periods. For example, if you set up a frequency capping rule of sending no more than one campaign a day, a user may receive a message at 11pm in their local time zone, and they would be eligible to receive another message an hour later.
```

## Email

> #### Things to check for:

Reviewer: ```suggestion
#### Things to check for
```
I think we can remove the colons here (similar heading to this [Things to know ](https://www.braze.com/docs/user_guide/message_building_by_channel/webhooks/creating_a_webhook/#things-to-know) section from the Creating a Webhook doc), but if we want to keep them for styling preferences then we should add colons to the two headings below "General"

- **Estimate segment from SMS copy**: Test your SMS copy in the [SMS segment calculator]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/segments/#things-to-keep-in-mind-as-you-create-your-copy). Keep in mind that the number of SMS segments should be taken into account with your throughput capabilities. (Audience * SMS segments = Throughput needed). Refer to SMS FAQs on [avoiding overages]({{site.baseurl}}/user_guide/message_building_by_channel/sms/faqs/#how-can-i-avoid-overages).
- **SMS laws and regulations**: [Review SMS laws, regulations, and abuse prevention]({{site.baseurl}}/user_guide/message_building_by_channel/sms/sms_laws_and_regulations/) to ensure that you are using the SMS services in compliance with all applicable laws. Make sure you should seek the advice of your legal counsel before sending.

#### Things to know about:
- **SMS message defaulting**: SMS messages are normally defaulted to be sent from the short code in the sender pool.
- **Alphanumeric sender ID**: Two-way messaging will no longer work if you use an alphanumeric sender ID; these are now one-way only.

> - **Updated throughput in the US**: Throughput has changed in the US with US [A2P 10DLC registration](https://support.twilio.com/hc/en-us/articles/1260803225669-Message-throughput-MPS-and-Trust-Scores-for-A2P-10DLC-in-the-US). Note that we do not commit to any sending speed SLAs contractually due to multiple factors such as traffic congestion, carrier issues, etc. that may impact the actual delivery rates.

Reviewer: ```suggestion
- **Updated throughput in the US**: Throughput has changed in the US with US [A2P 10DLC registration](https://support.twilio.com/hc/en-us/articles/1260803225669-Message-throughput-MPS-and-Trust-Scores-for-A2P-10DLC-in-the-US). Note that we do not commit to any sending speed SLAs contractually due to multiple factors such as traffic congestion and carrier issues that may impact the actual delivery rates.
```
